src/model/Book.tsx

```diff
@@ -1,7 +1,19 @@
 import Author from './Author'
 
-export default interface BookBody {
-    Id?: number
+export interface BookBody {
```

Comment:
if you have created a new files for this interface, remove it from here.

Reply:
OK. I remove it.